test/tools/ossfuzz/protoToYul.h

```diff
@@ -75,19 +80,90 @@ class ProtoConverter
 	void visit(RetRevStmt const&);
 	void visit(SelfDestructStmt const&);
 	void visit(TerminatingStmt const&);
+	void visit(FunctionCallNoReturnVal const&);
+	void visit(FunctionCallSingleReturnVal const&);
+	void visit(FunctionCall const&);
+	void visit(FunctionDefinitionNoReturnVal const&);
+	void visit(FunctionDefinitionSingleReturnVal const&);
+	void visit(FunctionDefinitionMultiReturnVal const&);
+	void visit(Program const&);
 	template <class T>
 	void visit(google::protobuf::RepeatedPtrField<T> const& _repeated_field);
+	void registerFunction(FunctionDefinition const&);
 
 	std::string createHex(std::string const& _hexBytes) const;
 	std::string createAlphaNum(std::string const& _strBytes) const;
 	bool isCaseLiteralUnique(Literal const&);
+	enum class NumFunctionReturns
+	{
+		None,
+		Single,
+		Multiple
+	};
+
+	template<class T>
+	void visitFunctionInputParams(T const&, int);
+
+	template<class T>
+	void createFunctionStub(T const&, int, int, NumFunctionReturns);
+	inline std::string functionTypeToString(NumFunctionReturns _type)
```

Comment:
Why `inline`? (`inline` only has a defined effect on free functions, compilers usually ignore this recommendation)
Also please move to .cpp file.

Reply:
Ah, okay.

Reply:
done